packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.js

```diff
@@ -25,7 +25,7 @@ const propTypes = {
   /** Additional classes added to the LoginPage. */
   className: PropTypes.string,
   /** Attribute that specifies the URL of the brand image for the LoginPage */
-  brandImgSrc: PropTypes.string,
+  brandImgSrc: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
wrong prop? Should be updating backgroundImgSrc. Also, instead of object would be better to use shape:
```
PropTypes.shape({
      xs: PropTypes.string,
      xs2x: PropTypes.string,
      sm: PropTypes.string,
      sm2x: PropTypes.string,
      lg: PropTypes.string,
      filter: PropTypes.string
    })
```